package.json

```diff
@@ -1,17 +1,28 @@
 {
-  "name": "databraid-react-starter",
+  "name": "transit-spa",
   "version": "1.0.0",
   "private": true,
   "dependencies": {
+    "enzyme": "^2.9.1",
+    "enzyme-to-json": "^1.5.1",
     "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "react-dom": "^15.6.1",
+    "redux": "^3.7.2",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.2.0",
+    "semantic-ui-react": "^0.71.3"
   },
   "devDependencies": {
-    "eslint": "4.3.0",
-    "eslint-config-airbnb": "15.1.0",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-jsx-a11y": "5.1.1",
-    "eslint-plugin-react": "7.1.0",
+    "babel-jest": "^20.0.3",
```

Comment:
what's the use for babel-jest, babel-presets, etc? I think these all come already with create-react app. I do think you need the eslint ones and react-test-renderer